packages/main/src/plugin/provider-registry.ts

```diff
@@ -1173,12 +1232,27 @@ export class ProviderRegistry {
     // grab the correct provider
     const provider = this.getMatchingProvider(internalProviderId);
 
-    // grab the correct kubernetes connection
+    // grab the correct inference connection
```

Comment:
nitpick: thanks for correcting this typo, but this is not in the scope of this PR